README.md

```diff
@@ -112,6 +120,16 @@ providers:
 
 ### OAuth2 Providers
 
+#### Jira
+
+|Key                   |Description                 | Values |
+|:---------------------|:---------------------------|:-------|
+|enabled|Enable or disable this specific provider. This stops its showing as an valid login option| [default: `true`] \| `false` |
```

Comment:
j'ai pas du tout compris `This stops its showing as an valid login option` ?

Reply:
J'ai fais un copier/coller du texte de présentation de github, et c'est le même peu importe le provider.
En gros, si le provider n'est pas activé, il ne sera pas présent sur la page de connexion